### DISCUSSION TITLE
3.10 quick find in vlans is not responding

### DISCUSSION THREAD

**Discussion by jcralbino** [netbox-community/netbox#7918] [Q&A] [answered]
Hello 

we are using netbox 3.10, and i am facing an issue within our deployment where we go to the search (quick find) this page stops responding and it show issues 
![image](https://user-images.githubusercontent.com/44149262/143194046-fdf944de-d213-4f51-ba45-183e1aec0bb7.png)

Most of the times querying in this search box breaks the responsiveness of the page .

Is there a way we can limit and control this behavior ?

**Comment by kkthxbye-code** [comment, marked as answer]
>Is there a way we can limit and control this behavior ?

Yes, lower the results per page at the bottom of the page. The feature filters the results, if you have the results per page set to a high number it demolishes performance. In general it's best to use the search bar or the filter instead, as the quick find only filters the current page results.

However I think the performance of the filter could be improved and creating an issue might be a good idea. The current code reflows the layout after processing every single element in the list, which is why it is incredibly slow.
